libs/SoloPlay/Tools/Tracker.js

```diff
@@ -14,6 +14,7 @@ const Tracker = {
 	SPPath: "libs/SoloPlay/Data/" + me.profile + "/" + me.profile + "-ScriptPerformance.csv",
 	tick: 0,
 	default: {
+		"Days": 0,
```

Comment:
Days is just being pulled from Total so it doesn't need its own declaration here in tracker
this goes for lines 73, 107, and 152, as well